### PR TITLE
WEB: Add greeting note to CoC

### PR DESCRIPTION
**Pull request by MarcoGorelli**
I noticed something similar in the [CPython contributing docs](https://cpython-core-tutorial.readthedocs.io/en/latest/diversity.html):

> For example, don’t say “hey guys!” but “hey everyone!”.

and thought it was really nice.

Pushing directly rather than opening as good first issue to avoid invoking a s*******m

### CODE REVIEW

**Comment by WillAyd** [comment]
Should be good - can you merge master to get green?

File: web/pandas/about/team.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ If you want to support pandas development, you can find information in the [dona
 > or anyone willing to increase the diversity of our team.
 > We have identified visible gaps and obstacles in sustaining diversity and inclusion in the open-source communities and we are proactive in increasing
 > the diversity of our team.
+> We welcome inclusive behavior - for example, please don’t say “hey guys!” but “hey everyone!”.
```
Comment (datapythonista) [inline web/pandas/about/team.md]: This is the team page, doesn't seem relevant to me here. Should it be in the code of conduct instead?

**Comment by jorisvandenbossche** [comment]
I agree this feels a bit "out of place" where it is added now

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @MarcoGorelli

**Comment by datapythonista** [comment]
If you can merge master, looks like something (I guess unrelated) failed.

**Comment by WillAyd** [comment]
Thanks @MarcoGorelli
